api/envoy/admin/v3/config_dump.proto

```diff
@@ -381,3 +381,20 @@ message EndpointsConfigDump {
   // The dynamically loaded endpoint configs.
   repeated DynamicEndpointConfig dynamic_endpoint_configs = 3;
 }
+
+// Envoy's admin fills this message with all currently known dynamic extension configurations.
+message ExtensionsConfigDump {
+  message DynamicExtensionConfig {
+    // This is the per-resource version information.
```

Comment:
Please add a reference to: envoy_api_field_service.discovery.v3.DiscoveryResponse.version_info

Reply:
Done, thanks!